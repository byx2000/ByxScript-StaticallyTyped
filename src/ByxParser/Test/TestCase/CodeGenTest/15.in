function int f1()
{
	int i = 12;
	return i;
}

function double f2()
{
	double x = 3.14;
	return x;
}

function void main()
{
	return;
}

function int f3()
{
	double x = 3.14;
	return x;
}

function double f4()
{
	int i = 12;
	return i;
}