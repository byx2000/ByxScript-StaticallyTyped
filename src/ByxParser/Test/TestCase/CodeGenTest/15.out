5
0 10
1 0
1 5
1 12
1 17

0

iconst 12
istore 0
iload 0
ret
ret

dconst 3.14
dstore 0
dload 0
ret
ret

ret
ret

dconst 3.14
dstore 0
dload 0
ret
ret

iconst 12
istore 0
iload 0
ret
ret